### PR TITLE
Add mobile: handlers for getting and setting clipboard content

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Depends on https://github.com/appium/appium-uiautomator2-server/pull/124

### CODE REVIEW

File: lib/commands/clipboard.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+commands.mobileSetClipboard = async function (opts = {}) {
+  if (!util.hasValue(opts.content)) {
+    log.errorAndThrow(`The 'content' argument is mandatory`);
+  }
+  return await this.uiautomator2.jwproxy.command('/appium/clipboard/set', 'POST', opts);
+};
+
+commands.mobileGetClipboard = async function (opts = {}) {
+  return await this.uiautomator2.jwproxy.command('/appium/clipboard/get', 'POST', opts);
```
Comment (dpgraham) [inline lib/commands/clipboard.js]: How the `/get` endpoint method uses `POST` and not `GET`?

Comment (jlipps) [reply lib/commands/clipboard.js]: even so, could we include that as a url param like `/appium/clipboard/get/:contentType`?

Comment (mykola-mokhnach) [reply lib/commands/clipboard.js]: I find POST more flexible %)

Comment (dpgraham) [reply lib/commands/clipboard.js]: Can it not use query strings? (e.g.: `/appium/clipboard?contentType=<...>&...`

**Comment by mykola-mokhnach** [comment]
I've decided to unify this with iOS and put the endpoint into base driver. Will update this PR as soon as base driver changes are available.

**Pull request closed without merging**